srcpkgs/backintime/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'backintime'
 pkgname=backintime
-version=1.1.24
+version=1.2.0
 revision=1
 archs=noarch
 build_wrksrc="common"
 build_style=configure
 configure_args="--python3"
-hostmakedepends="python3"
-depends="python3 rsync python3-dbus"
 pycompile_dirs="/usr/share/backintime"
-short_desc="A simple backup tool for Linux"
-maintainer="Juan RP <xtraeme@voidlinux.org>"
+hostmakedepends="python3"
+depends="python3 rsync python3-dbus python3-keyring"
+short_desc="Simple backup tool for Linux"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
```

Comment:
Juan recently resigned to maintain packages. Do you want to take it over?

Also `voidlinux.eu` is deprecated in favor of `.org`.